iOS_Concepts.playground/Contents.swift

```diff
@@ -1,4 +1,21 @@
 import UIKit
 
+func Fibonacci(numSteps: Int)->[Int] {
```

Comment:
This function is incorrect it doesnt print a fibonacii series

Reply:
i have fixed the issue. The function now correctly prints a Fibonacci series.